### PR TITLE
Try to fix flaky 02210_processors_profile_log

### LINKED ISSUE #50996: 02210_processors_profile_log is flaky
https://play.clickhouse.com/play?user=play#U0VMRUNUIHB1bGxfcmVxdWVzdF9udW1iZXIsIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgdGVzdF9zdGF0dXMsIGNoZWNrX3N0YXR1cywgcmVwb3J0X3VybApGUk9NIGNoZWNrcwpXSEVSRSAxCiAgICBBTkQgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnT0snCiAgICBBTkQgY2hlY2tfc3RhdHVzICE9ICdzdWNjZXNzJwogICAgQU5EIHRlc3RfbmFtZSBsaWtlICclMDIyMTBfcHJvY2Vzc29yc19wcm9maWxlX2xvZyUnCk9SREVSIEJZIGNoZWNrX3N0YXJ0X3RpbWUgZGVzYywgY2hlY2tfbmFtZSwgdGVzdF9uYW1lCg==

https://s3.amazonaws.com/clickhouse-test-reports/0/8f9c74debb002b74180ad534e828714b49bba44a/stateless_tests__release_.html

https://s3.amazonaws.com/clickhouse-test-reports/0/cd3c4475a941ab3b30c0f366b4ba961ef26b00de/stateless_tests__release_.html

https://s3.amazonaws.com/clickhouse-test-reports/0/8dddec4c4cfa00d6d27a0785dd63bdae7d885b53/stateless_tests__release__s3_storage__[1_2].html

### PR DESCRIPTION
**Pull request by devcrafter**
Fixes #50996

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
This is an automated comment for commit a705b08bd81658e878d7b7d214b057c661bbed69 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51641/a705b08bd81658e878d7b7d214b057c661bbed69/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

**Comment by qoega** [comment]
https://github.com/ClickHouse/ClickHouse/issues/51648

File: tests/queries/0_stateless/02210_processors_profile_log.reference [hunk 1]
```diff
@@ -18,13 +18,13 @@ SELECT
...
-        name = 'ExpressionTransform', elapsed_us>1e6,
+        name = 'ExpressionTransform', elapsed_us>=1e6,
         -- SourceFromSingleChunk, that feed data to ExpressionTransform,
         -- will feed first block and then wait in PortFull.
-        name = 'SourceFromSingleChunk', output_wait_elapsed_us>1e6,
+        name = 'SourceFromSingleChunk', output_wait_elapsed_us>=1e6,
         -- NullSource/LazyOutputFormatLazyOutputFormat are the outputs
         -- so they cannot starts to execute before sleep(1) will be executed.
-        input_wait_elapsed_us>1e6)
+        input_wait_elapsed_us>=1e6)
```
Comment (tavplubix) [inline tests/queries/0_stateless/02210_processors_profile_log.reference:27]: Still flaky: https://s3.amazonaws.com/clickhouse-test-reports/0/6c51b771c57ffd597398aa24b96fa68d527b75b7/stateless_tests__release_.html

Comment (azat) [reply tests/queries/0_stateless/02210_processors_profile_log.reference:27]: https://github.com/ClickHouse/ClickHouse/pull/66239

**Pull request merged**